lib/calendar.js

```diff
@@ -60,7 +60,7 @@ export default class Calendar {
     } else {
       query = this.xcodeVersion.major < 10 ?
         `INSERT INTO 'access' VALUES ('kTCCServiceCalendar', '?', 0, 1, 1, 0, 0);` :
-        `INSERT INTO 'access' VALUES ('kTCCServiceCalendar', '?', 0, 1, 1, 0, 0, 0, 0, 0);`;
+        `INSERT INTO 'access' VALUES ('kTCCServiceCalendar', '?', 0, 1, 1, 0, 0, 0, 0, 0, 0);`;
```

Comment:
is this safe for previous DB versions?

Reply:
I don't think that it matters, for betas. There is no reason to support a previous beta version of Xcode.